chef/cookbooks/barclamp/libraries/barclamp_library.rb

```diff
@@ -201,7 +201,7 @@ def self.lookup_interface_info(node, conduit, intf_to_if_map = nil)
         return [nil, nil] if intf_to_if_map[conduit].nil?
 
         c_info = intf_to_if_map[conduit]
-        interface_list = c_info["if_list"]
+        interface_list = c_info["if_list"] || []
```

Comment:
Wouldn't we crash a few lines later if it's `nil` (when we do `interface_list.size == 1`)?